client/me/purchases/manage-purchase/index.jsx

```diff
@@ -75,7 +75,7 @@ import QueryCanonicalTheme from 'components/data/query-canonical-theme';
 import QueryUserPurchases from 'components/data/query-user-purchases';
 import RemovePurchase from '../remove-purchase';
 import VerticalNavItem from 'components/vertical-nav/item';
-import paths from '../paths';
+import { cancelPurchase, cancelPrivacyProtection, purchasesRoot } from '../paths';
```

Comment:
Nit: I've noticed many of us sort the imports in alphabetical order, this seems like a good place to do it too if you wish.

Reply:
Good catch! Sorted in 5377a8cf4c